fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -67,21 +100,28 @@ const FeastSearchContainerComponent = ({
     <React.Fragment>
       <InputContainer>
         <TextInputContainer>
-          <TextInput placeholder="Search recipes" displaySearchIcon />
+          <TextInput
+            placeholder="Search recipes"
```

Comment:
It might be good for this placeholder to vary according to whether the chefs or recipes toggle is selected. Not sure whether that should be part of this PR or could come later, though!